src/modules/esl-event-listener/core/targets/swipe.target.ts

```diff
@@ -73,17 +72,18 @@ export class ESLSwipeGestureTarget extends SyntheticEventTarget {
   @bind
   protected handleStart(e: PointerEvent): void {
     this.startEvent = e;
-    this.isGestureStarted = true;
+    this.target.setPointerCapture && this.target.setPointerCapture(e.pointerId);
   }
 
   /**
    * @param e - pointer event (pointerdown)
    * @returns diff between pointerdown and pointer coordinates and timestamp {@link EventsDiff}
    */
   protected eventDiff(e: PointerEvent): EventsDiff {
+
     return {
-      x: this.startEvent.clientX - e.clientX,
-      y: this.startEvent.clientY - e.clientY,
+      x: Math.round(e.clientX - this.startEvent.clientX),
+      y: Math.round(e.clientY - this.startEvent.clientY),
       time: e.timeStamp - this.startEvent.timeStamp
     };
```

Comment:
Just a couple of ideas:
 - We can make EventsDiff compatible with ESLSwipeGestureEventInfo
 - Optionally additional calculations (angle, distance) can be concentrated here.
 - Possibly the most readable case will be if the `handleEnd` will operate the following steps and methods:
    - calculate vector (which is EventsDiff) + direction
    - condition function to filter too small or to slow vectors (timeout, distance restrictions from resolveDirection)
    - in case EventsDiff is compatible with ESLSwipeGestureEventInfo we get super short and clear method as the result